src/react-components/room/PeopleSidebar.js

```diff
@@ -101,7 +149,8 @@ export function PeopleSidebar({
   voiceChatEnabled,
   isMod
 }) {
-  const intl = useIntl();
+  const user = people.find(person => !!person.isMe);
```

Comment:
I think this all could work fine, another take that might make the jsx less confusing is to alter the people array, something like this.  

```
const me = people.find(person => !!person.isMe);
filteredPeople = people.filter(person => !person.isMe);
filteredPeople.unshift(me);
```

and then you can just loop through filteredPeople